pages/documentation/settings.md

```diff
@@ -19,17 +21,37 @@ subnav:
+## Introducing USWDS settings
+USWDS is built using a suite of customizable settings that allows you to configure the Design System to fit your project's needs. These settings are Sass variables that begin with the `$theme-` prefix and are typically defined with [USWDS design tokens]({{ site.baseurl }}/design-tokens). You can find the full list of USWDS settings and their default values in the [USWDS settings tables](#general-settings).
+
+## Configuring custom USWDS settings
+All USWDS settings are defined in the `uswds-core` package and can be configured using Sass' [`@use..with()`](https://sass-lang.com/documentation/at-rules/use#configuration) rule. To create a custom configuration of USWDS, add the USWDS settings variables that you wish to modify inside the parentheses of this statement, as shown in this example:
 
 ```scss
-@forward 'uswds-theme';
-@forward 'uswds';
-@forward 'uswds-theme-custom-styles';
+  @use "uswds-core" with (
+    $theme-image-path: "../uswds/img",
+    $theme-show-compile-warnings: true,
+    $theme-show-notifications: false,
+    $theme-banner-background-color: "ink",
+    $theme-banner-link-color: "primary-light",
+    $theme-banner-max-width: "widescreen",
+    $theme-checkbox-border-radius: "md",
+  )
 ```
+Note that the settings variables in this module inform both general theme and component customizations.
+
+### What to include in your configuration
+Include only USWDS settings that you wish to modify in this configuration. These items should be comma-separated and given values of the appropriate type, as identified in the tables below.
 
-Add only the settings you wish to modify before importing `uswds`, either in a separate file (`_uswds-theme.scss`), or directly in your Sass entry point. Either way you do it, you'll use the format `@use "uswds-core" with ()` including a list of your changed USWDS settings variables inside the parentheses.
+Please note that this configuration accepts only current USWDS settings variables &mdash; adding anything else here will result in an error.
 
-If you use the `_uswds-theme.scss` file, it would look something like this:
+### Where to include your configuration
+Full instructions for setting up Sass files for USWDS can be found in [Phase 2 of the Getting Started for Developers]({{ site.baseurl }}/documentation/getting-started/developers/phase-two-compile/#step-1-set-up-your-projects-sass-entry-point) guide.
+
+The important thing to note is that your settings configuration must be loaded **_before_ including the main USWDS Sass**. We've created some example setups to give you an idea of how this works:
```

Comment:
Line 51: I'm a bit confused about the order of operations, here. Where are the steps to include the main USWDS sass? I feel a bit nervous that if I get to this step I might have already included the main uswds sass without knowing/accidentally. Could we use active voice and make the order very clear.  Say something like, 

To make sure everything imports properly:

1. load your settings configuration (hyperlinked)
2. include the main Uswds sass (hyperlinked if appropriate)

We've created some example setups... [...]

[optional: explain what error they could see if they get this order wrong in case people are searching the site by error]